### DISCUSSION TITLE
Welcome to the Community. ! - Start Here

### DISCUSSION THREAD

**Discussion by omarcostahamido** [Quantumland-art/community#5] [Announcements]
Welcome to the **Community.**
A place to share music, books, software, films, events, and ideas dealing with Quantum Computing (QC) and Creative Practice.
In an effort to bring together creative artists and other interested parties from all corners of the globe, this is an online-first community. It is hosted on Github, accessible to everyone, and as such, it also self-maintained and open to contributions. 
Ultimately, this should be a space to promote new interactions and exchange, between Art and Science.

The discussion categories are as follows:
[📣 Announcements](https://github.com/orgs/Quantumland-art/discussions/categories/announcements): Where announcements about the platform will be shared by repository maintainers
[📍 Events](https://github.com/orgs/Quantumland-art/discussions/categories/events): Share what is happening (online or in person). Conferences, music festivals, residencies, exhibitions, hackathons, etc.
[💬 General](https://github.com/orgs/Quantumland-art/discussions/categories/general): Chat about anything and everything related to QC + creative practice, and **Community.** building!
[😃 People](https://github.com/orgs/Quantumland-art/discussions/categories/people): A good place to start. Introduce yourself, and get to know the **Community.**
[📚 Bookshelf](https://github.com/orgs/Quantumland-art/discussions/categories/bookshelf): Share books, articles, and other writings.
[💽 Music](https://github.com/orgs/Quantumland-art/discussions/categories/music): Share Music albums and new works.
[💻Tech](https://github.com/orgs/Quantumland-art/discussions/categories/tech): Share new software and digital instruments.
[🎬Video](https://github.com/orgs/Quantumland-art/discussions/categories/video): Share Films, docs, videos, and new event recordings.

Please note that in order to make a new post, or reply to a current thread, you will need to have a [free Github account](https://community.quantumland.art/faq/github-account).
In order to directly contribute to the code base of the **Community.** webpage, please visit [the issues tab](https://github.com/Quantumland-art/community/issues) of the community repository. 
Finally, don't forget to enable notifications by watching the [community repo](https://github.com/Quantumland-art/community) (as explained [here](https://github.com/orgs/Quantumland-art/discussions/64)).

Best,
OCH

[community.quantumland.art](https://community.quantumland.art)

**Comment by medusamedusa** [comment]
Hello! I would like to share this blog post about the symposium. It is in Italian; the English translation will probably appear the next month. The venue is a very active blog of applied mathematics. https://www.mathisintheair.org/wp/2023/10/quantum-music-a-berlino/

**Reply by omarcostahamido** [reply]
@medusamedusa super!
can you share this as a new post under the [📚 Bookshelf](https://github.com/orgs/Quantumland-art/discussions/categories/bookshelf) category? Otherwise only people that look under the replies of this thread will find it 😄 
(note to self: I need to create a more thorough set of instructions on `how to use the forum` 😅 )
Thank you 🙏
